src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/stringreturners/BuzzStringReturner.java

```diff
@@ -1,11 +1,16 @@
 package com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.impl.stringreturners;
 
+import com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.impl.factories.BuzzModelFactory;
+import com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.factories.ModelFactory;
+import com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.models.Model;
 import com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.stringreturners.StringStringReturner;
 
 public class BuzzStringReturner implements StringStringReturner {
 
 	public String getReturnString() {
-		final StringBuilder myStringBuilder = new StringBuilder("Buzz");
+		final ModelFactory factory = new BuzzModelFactory();
+		final Model model = factory.createModel();
+		final StringBuilder myStringBuilder = new StringBuilder(model.getValue());
```

Comment:
This seems flaky - what if we want to replace the StringBuilder implementation? Perhaps a factory is in order